zerver/tests/test_import_export.py

```diff
@@ -428,3 +442,104 @@ def get_set(data: List[Dict[str, Any]], field: str) -> Set[str]:
 
         exported_messages_recipient = get_set(messages['zerver_message'], 'recipient')
         self.assertIn(list(exported_messages_recipient)[0], exported_recipient_id)
+
+    """
+    Start tests for import
+    """
+    def test_import_realm(self) -> None:
+
+        realm = Realm.objects.get(string_id='zulip')
+        RealmEmoji.objects.get(realm=realm).delete()
+        self._export_realm(realm)
+
+        with patch('logging.info'):
+            do_import_realm('var/test-export', 'test-zulip')
+
+        # sanity checks
+
+        # test realm
+        self.assertTrue(Realm.objects.filter(string_id='test-zulip').exists())
+        imported_realm = Realm.objects.get(string_id='test-zulip')
+        self.assertNotEqual(imported_realm.id, realm.id)
+
+        # test users
+        realm_user_emails = set([
+            user.email for user in realm.get_admin_users()])
+        imported_realm_user_emails = set([
+            user.email for user in imported_realm.get_admin_users()])
+        self.assertEqual(imported_realm_user_emails, realm_user_emails)
+        realm_user_emails = set([
+            user.email for user in realm.get_active_users().filter(is_bot=False)])
+        imported_realm_user_emails = set([
```

Comment:
To continue the previous point, with so many of these having an `imported_` prefix, I wonder if it might be more succinct to have something like:
```python
realms = [original_realm, imported_realm]  # changing the variables in use to avoid re-use
realm_user_emails = [{user.email for user in realm.get_admin_users()} for realm in realms]
self.assertEqual(realm_user_emails[0], realm_user_emails[1])
```
The latter could be made clearer with a `namedtuple` instead of a list, to enable `.original` and `.imported`.

Reply:
This does look better. I'll update it.